packcheck.sh

```diff
@@ -1311,15 +1311,8 @@ create_and_unpack_pkg_dist() {
       local pi_files_exist=""
       local pi_files_n_exist=""
       while read p; do
-          if test -f "$p"
-          then
-              pi_files_exist="$p\n$pi_files_exist"
-          else
-              if test -n "$p"
-              then
-                  pi_files_n_exist="$p\n$pi_files_n_exist"
-              fi
-          fi
+          x=`git ls-files $p`
+          pi_files_exist="$x\n$pi_files_exist"
```

Comment:
1. The comment at line 1310 is not accurate now.
2. pi_files_n_exist remains blank so the warning at line 1319 will never come.